Woof/WoofTests/Model/Walking/WalkingStatusTests.swift

```diff
@@ -0,0 +1,13 @@
+import XCTest
+
+final class WalkingStatusTests: XCTestCase {
+    func testDescriptionReturnsExpectedStringAndExist() {
```

Comment:
Can you add the test to check all statuses are presented?

Reply:
Added with 3239009